### ISSUE TITLE
Unresolved reference when adding dependency using compileOnlyAar 

### ISSUE THREAD

**Issue by alexsurinov** [stepango/aar2jar#2]
Thanks for the plugin!

I try to use this plugin to add rxAndroid `aar` dependency to my pure Kotlin gradle module.
This is a build.gradle file of my Kotlin module:
```gradle
plugins {
    id 'kotlin'
    id 'kotlin-kapt'
    id 'com.stepango.aar2jar' version "0.4"
}

dependencies {
    def dependencies = rootProject.ext.dependencies

    // Core app modules.
    implementation project(':core-di')

    // 3rd-party libs.
    implementation dependencies.kotlin
    implementation dependencies.rxJava
    compileOnlyAar dependencies.rxAndroid
    compileOnly androidJar

    // Kotlin annotation processors.
    kapt dependencies.daggerCompiler

    // Test libs.
    testImplementation dependencies.junit
}
```
My project builds successfully, but there is a dependency resolving issue when I try to import the library class:
```kotlin
// Unresolved reference here
import io.reactivex.android.schedulers.AndroidSchedulers
```
Could you explain what's the problem and how to handle this issue?

**Comment by DmitriyG1** [comment]
Same error. I tried to use it through "compileOnlyAar" and "implementationAar"

**Comment by stepango** [comment]
Will be fixed next release.
Workaround for now is 

```
plugins {
    id 'kotlin'
    id 'idea'
    id 'com.stepango.aar2jar' version "0.4"
}

dependencies {
    implementation fileTree(dir: 'libs', include: ['*.jar'])

    compileOnly androidJar
    compileOnlyAar "androidx.lifecycle:lifecycle-extensions:2.0.0"
    compileOnlyAar "androidx.lifecycle:lifecycle-livedata-core:2.0.0"
    compileOnlyAar 'androidx.appcompat:appcompat:1.0.2'
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
}

afterEvaluate {
    idea {
        module {
            scopes.forEach { name, map ->
                map.plus += [configurations.compileOnlyAar]
            }
        }
    }
}
```

**Comment by stepango** [comment]
Fixed in v0.5 by https://github.com/stepango/aar2jar/commit/fc4d026e17b14c2e02a1dfb89a46dce623f0a6f6
